src/components/Sourcing/PaStageManufacturer.jsx

```
@@ -62,218 +62,124 @@ export default function PaStageManufacturer(props) {
-                                ? "negative"
-                                : ""
-                        }`}
-                      >
-                        {pa.profit === 0
-                          ? pa.profit?.toLocaleString()
-                          : pa.profit?.toLocaleString()}
-                      </div>
-                    </div>
-                  </div>
-                )
+                   <div className="row border-bottom py-2 align-items-center">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)